packages/rancher-logging/generated-changes/overlay/templates/loggings/rke2/daemonset.yaml

```diff
@@ -18,9 +18,17 @@ spec:
       containers:
         - name: fluentd
           image: "{{ template "system_default_registry" . }}{{ .Values.images.fluentbit.repository }}:{{ .Values.images.fluentbit.tag }}"
+          {{- if .Values.global.seLinux.enabled }}
+          securityContext:
+            seLinuxOptions:
+              type: rke_logreader_t
+          {{- end }}
           volumeMounts:
             - mountPath: /fluent-bit/etc/
               name: config
+            - mountPath: /run/log/journal
+              name: journal
+              readOnly: true
```

Comment:
Should we wrap this with seLinux.enabled? I'm leaning towards "no", but wanted to ask.

Reply:
no we need to mount it for non se envs too